### DISCUSSION TITLE
List Size Options Hotkey

### DISCUSSION THREAD

**Discussion by rjemanuele** [rxhanson/Rectangle#316] [Ideas]
So I suck at remembering a ton of shortcut keys but several stick in my head readily.  I'd like a shortcut key to bring up a list of window sizing options I can scroll through with the arrow keys.

Tools like Apple Spotlight and Boop stick in my head for good examples of this.  In those I can access a wealth of features with keystrokes and never need to move the mouse even I cannot remember a specific hotkey, I have access to the results of master hotkey.

**Comment by rxhanson** [comment]
That makes sense to me. The simplest approach would be to allow a shortcut to open the status bar menu, but that's kind of a lot of options to arrow through depending on what you want. It would be nice to bring up just a context menu wherever the cursor is with a list defined by the user. I don't have the bandwidth to add this in at this time, but would definitely consider a PR for it, in whichever form it might come in.
